apps/portal/src/app/engine/features/backend-wallets/page.mdx

```diff
@@ -12,6 +13,50 @@ Engine performs blockchain actions using backend wallets that you own and manage
 
 There are multiple options for securing backend wallets.
 
+## Smart Backend Wallets
+
+Smart backend wallets are the recommended way to perform blockchain operations with Engine. Each smart backend wallet consists of an EOA (managed internally by Engine) and a smart account (using thirdweb's default account implementation).
+
+### Benefits
+
+Smart backend wallets inherit smart account benefits and offer several advantages over traditional EOAs:
+
+- **Gas Management**: Built-in paymaster eliminates the need to maintain gas tokens. This means you never need to hold crypto or top up gas.
+- **Better Nonce Management**: Smart accounts use multi-dimensional nonces, which are more efficient than EOAs.
+- **Simple Integration**: Works with all existing Engine endpoints without any code changes.
+
+### How it works
+
+The smart account is automatically deployed the first time you send a transaction on a chain. You don't need to think about deploying or managing the smart account - Engine handles everything behind the scenes. All transactions are sent as UserOperations to the EntryPoint contract on chain.
+
+<Details summary="How are Smart Backend Wallets different from existing engine AA features?">
+	Unlike the [previous account abstraction implementation in
+	Engine](account-abstraction) where you had to manage both the account address
+	(`x-account-address`) and the backend wallet address
+	(`x-backend-wallet-address`) separately, smart backend wallets simplify this.
+	The backend wallet address is now the smart account address itself.
+</Details>
+
+<Details summary="When not to use Smart Backend Wallets?">
```

Comment:
Hm would someone need this, or would we build this in the future? Maybe this just belongs in an FAQ section at the bottom:

```markdown
## FAQ

#### Can I import a smart account?
No, importing a smart account is currently not supported.
```